src/sub.c

```diff
@@ -509,7 +509,7 @@ DPS_Status DPS_SendSubscriptionAck(DPS_Node* node, RemoteNode* remote, int colli
     default:
         return DPS_ERR_INVALID;
     }
-    if (remote->outbound.sendInterests) {
+    if (flags & DPS_SUB_FLAG_SAK_REQ) {
```

Comment:
I can't appear to comment on other lines in the file...will numMapEntries be correct when collision && (flags & DPS_SUB_FLAG_SAK_REQ) (see line 487)?

Reply:
Yes, the flag is what the receiving end keys off to decide if interests are expected in the SAK.